setup.py

```diff
@@ -1,21 +1,20 @@
 from setuptools import setup, find_packages
 from os import path
 
-pkg_name = 'element_array_ephys'
+pkg_name = 'element-array-ephys'
 here = path.abspath(path.dirname(__file__))
 
-long_description = """"
-DataJoint Element for Extracellular Array Electrophysiology with Neuropixels probe
-"""
+with open(path.join(here, 'README.md'), 'r') as f:
+    long_description = f.read()
 
 with open(path.join(here, 'requirements.txt')) as f:
     requirements = f.read().splitlines()
 
-with open(path.join(here, pkg_name, 'version.py')) as f:
+with open(path.join(here, pkg_name.replace('-', '_'), 'version.py')) as f:
     exec(f.read())
 
 setup(
-    name='element-array-ephys',
+    name=pkg_name,
```

Comment:
```suggestion
    name=pkg_name.replace('_', '-'),
```

You can also use an f-string for the `url` keyword below with the appropriate package name.